### ISSUE TITLE
Here is my need

### ISSUE THREAD

**Issue by elielnfinic** [NdekoCode/canvas-editor#4]
Image that we have 3 files(res/image_1.png, res/image_2.png and res/image_3.png) and a big text "Hello world", small text "Welcome to my world".

```
const img1 = "res/image_1.png";
const img2 = "res/image_2.png";
const img3 = "res/image_3.png";

cont txt_big = "Hello world";
const txt_small = "Welcome to my world";
```

TODO :

```
const get_processed_img = async (img1, img2, img3, txt_big, txt_small) => {
// SOME CODE
return processed_image;
}
```
You can use `sharp`, `canva` or `jimp` or any other library.

There is no need for HTML.

**Comment by NdekoCode** [comment]
The goal is to mix several images in one canvas and write as much text as you want ?

**Comment by elielnfinic** [comment]
Just some text like "Hello world". The artifact should be a simple JS function.

**Comment by NdekoCode** [comment]
I just added a function called `get_processed_img()`, This function allows to create an image and text using the HTML Canvas tag. I use the Jimp library to manipulate the images and texts. The images are loaded in parallel to improve performance, then drawn on the Jimp canvas according to the positions specified in the parameters. If the "contain" property is set to "true", images will be resized to fit a specified size to avoid cropping or loss of proportion. Text is drawn using user-specified fonts. Finally, the canvas is exported as a final image. To add a custom font, you can use Jimp's "loadFont" method to load the font from a file and use it with the "print" method to add text to the image.
The function `get_processed_img()`, must be executed in a node.js interface and not in the browser.

**Comment by elielnfinic** [comment]
I tried it, I can't make it work.

**Comment by NdekoCode** [comment]
What is the error when you execute it?

**Comment by elielnfinic** [comment]
<img width="1506" alt="Screenshot 2023-04-17 at 9 07 03 PM" src="https://user-images.githubusercontent.com/46634774/232586515-96cc8d35-5a65-4fb9-97fe-66b3e3c6585e.png">

**Comment by NdekoCode** [comment]
Non, j'ai enlever la partie HTML et créer juste une fonction simple, il faut juste faire un pull des nouvelles modification que je viens d'aborder, puis exécuter la fonction avec avec node.js avec la commande `node assets/js/get_processed_img.js`

**Comment by NdekoCode** [comment]
Non, je n'ai pas utilisé la partie HTML, il faut d'abord faire un pull request pour avoir les nouvelles modification puis d'exécuter la fonction avec node.js avec la commande  `node assets/js/get_processed_img.js`, s'il y a une erreur faite le moi savoir.

**Comment by elielnfinic** [comment]
Je comprend, la partie HTML m'a mis en confusion. Bon, je viens d'essayer de faire cela, je vais te partager le repo bientot.

**Comment by elielnfinic** [comment]
Try this out https://github.com/elielnfinic/unimage.git

**Comment by NdekoCode** [comment]
I just made a Pull request on it
